needle-zendesk-connector/src/server/api/routers/connectors-router.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  ConnectorRequestSchema,
+  CreateConnectorRequestSchema,
+  ZendeskResponseSchema,
+} from "~/models/connectors-models";
+import { createTRPCRouter, procedure } from "~/server/api/trpc";
+import {
+  createZendeskConnector,
+  deleteZendeskConnector,
+  getZendeskConnector,
+  listZendeskConnectors,
+  runZendeskConnector,
+} from "~/server/backend/connectors-backend";
+import { createZendeskService } from "~/server/zendesk/service";
+
+export const connectorsRouter = createTRPCRouter({
+  create: procedure
+    .input(CreateConnectorRequestSchema)
+    .mutation(async ({ ctx, input }) =>
+      createZendeskConnector(input, ctx.session),
+    ),
+
+  list: procedure.query(async ({ ctx }) => listZendeskConnectors(ctx.session)),
+
+  get: procedure
+    .input(ConnectorRequestSchema)
+    .query(async ({ ctx, input }) => getZendeskConnector(input, ctx.session)),
+
+  delete: procedure
+    .input(ConnectorRequestSchema)
+    .mutation(async ({ ctx, input }) =>
+      deleteZendeskConnector(input, ctx.session),
+    ),
+
+  run: procedure
```

Comment:
I have had this endpoint published for debugging reasons. In practice user should not be able to run a connector manually thought - can we remove it?

Reply:
just to confirm do you mean remove, run fn? Then run button should be removed from the UI